modules/zookeeper/src/test/java/org/apache/ignite/spi/discovery/zk/internal/ZookeeperDiscoverySegmentationAndConnectionRestoreTest.java

```diff
@@ -254,15 +253,12 @@ public void testSegmentation3() throws Exception {
             srvs.get(0).stop();
             srvs.get(1).stop();
 
-            QuorumPeer qp = srvs.get(2).getQuorumPeer();
-
-            // Zookeeper's socket timeout [tickTime * initLimit] + 5 additional seconds for other logic
-            assertTrue(l.await(qp.getTickTime() * qp.getInitLimit() + 5000, TimeUnit.MILLISECONDS));
+            assertTrue(l.await(30, TimeUnit.MILLISECONDS));
```

Comment:
Could you please clarify why this code was changed?

Reply:
Of course 30 milliseconds is not enough. Increased to 60 seconds. I think it is not necessary to calculate the timeout in any special way like `qp.getTickTime() * qp.getInitLimit() + 5000`.